test/metabase/api/search_test.clj

```diff
@@ -609,3 +611,34 @@
         (is (= ["Another SQL query" "Dataset"]
                (->> (search-request-data :rasta :q "aggregation")
                     (map :name))))))))
+
+(deftest search-db-call-count-test
+  (t2.with-temp/with-temp [:metabase.models.card/Card           _              {:name "card db count test 1"}
+                           :metabase.models.card/Card           _              {:name "card db count test 2"}
+                           :metabase.models.card/Card           _              {:name "card db count test 3"}
+                           :metabase.models.dashboard/Dashboard _              {:name "dash count test 1"}
+                           :metabase.models.dashboard/Dashboard _              {:name "dash count test 2"}
+                           :metabase.models.dashboard/Dashboard _              {:name "dash count test 3"}
+                           :metabase.models.database/Database   {db-id :id}    {:name "database count test 1"}
+                           :metabase.models.database/Database   _              {:name "database count test 2"}
+                           :metabase.models.database/Database   _              {:name "database count test 3"}
+                           :metabase.models.table/Table         {table-id :id} {:db_id  db-id
+                                                                                 :schema nil}
+                           :metabase.models.metric/Metric        _             {:table_id table-id
+                                                                                :name     "metric count test 1"}
+                           :metabase.models.metric/Metric        _             {:table_id table-id
+                                                                                :name     "metric count test 1"}
+                           :metabase.models.metric/Metric        _             {:table_id table-id
+                                                                                :name     "metric count test 2"}
+                           :metabase.models.segment/Segment      _             {:table_id table-id
+                                                                                :name     "segment count test 1"}
+                           :metabase.models.segment/Segment      _             {:table_id table-id
+                                                                                :name     "segment count test 2"}
+                           :metabase.models.segment/Segment      _             {:table_id table-id
+                                                                                :name     "segment count test 3"}]
```

Comment:
You should really just use the model name directly like `Dashboard` and not the full keyword. The full keyword is way too long to type. Maybe in the future we'll choose different shorter keywords like `:model/Dashboard` but for now just use `Dashboard`

Reply:
gotcha, I thought we were going to deprecate that stuff.